app/src/main/res/layout/state_fragment.xml

```diff
@@ -33,10 +25,43 @@
       android:layout_marginBottom="8dp"
       android:onClick="@{(v) -> stateFragment.dummyButtonClicked()}"
       android:text="Dummy Audio Button"
+      android:visibility="gone"
       app:layout_constraintBottom_toBottomOf="parent"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toTopOf="parent"
-    />
+      app:layout_constraintTop_toTopOf="parent"/>
+    <TextView
+      android:id="@+id/dummy_state_name"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="8dp"
+      android:layout_marginTop="8dp"
+      android:layout_marginEnd="8dp"
+      android:text="@{`State Name: `+viewModel.stateName}"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"/>
+    <androidx.recyclerview.widget.RecyclerView
+      android:id="@+id/state_recycler_view"
+      android:layout_width="0dp"
+      android:layout_height="0dp"
+      android:divider="@android:color/transparent"
+      android:dividerHeight="8dp"
+      android:padding="16dp"
+      app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent" app:layout_constraintTop_toBottomOf="@+id/dummy_state_name"/>
```

Comment:
Please wrap attributes rather than putting them on the same line.

Reply:
Resolved.